cypress/integration/plugins/security-dashboards-plugin/aggregation_view.js

```diff
@@ -69,6 +69,23 @@ if (Cypress.env('SECURITY_ENABLED') && Cypress.env('AGGREGATION_VIEW')) {
       cy.contains('a', 'se*');
     });
 
+    it('should check the saved objects by applying filter', () => {
+      CURRENT_TENANT.newTenant = 'global';
+      cy.visit(SAVED_OBJECTS_PATH);
+      cy.contains('a', 'Saved objects');
+
+      cy.get('span[title="Tenant"]').first().click({ force: true });
+      cy.get('span').contains('Private').click();
+      cy.contains('a', 's*');
+      cy.contains('a', 'se*').should('not.exist');
+
+      cy.wait(3000);
```

Comment:
We shall have a follow up issue to optimize the wait https://docs.cypress.io/guides/references/best-practices#Unnecessary-Waiting

Reply:
Got it, I will create a follow up for optimize those `cy.wait()`. Thanks for the reminder!